src/Verifier/SAW/Simulator/Concrete.hs

```diff
@@ -226,6 +226,9 @@ prims =
   , Prims.bpIntMax = pure2 max
   }
 
+unsupportedConcretePrimitive :: String -> a
+unsupportedConcretePrimitive = Prim.unsupportedPrimitive "Concrete"
+
```

Comment:
To make this mesh well with the previous change, I'd make the "C" lowercase.

Reply:
👍